src/EnvironmentAwareness.php

```diff
@@ -14,29 +15,48 @@
  */
 class EnvironmentAwareness implements TemplateGlobalProvider
 {
+    use Configurable;
+
+    public const ENV_LABEL = 'SS_ENVIRONMENT_LABEL';
+
     public static function getEnvironments()
     {
-        return Config::inst()->get('EnvironmentAwareness', 'environments');
+        // Legacy fallback
+        $envs = Config::inst()->get('EnvironmentAwareness', 'environments');
```

Comment:
Turns out this fallback doesn't quite cut it, because in this case you will only get the user defined config, it won't be merged with the config from the module.

Reply:
Might be worth adding a `getEnvironmentAwarenessConfig($property)` function that takes care of merging any legacy config on top of the namespaced/regular config?

To explain more, the issue is that someone might have provided for example just this:
```
EnvironmentAwareness:
  environments:
    Beta:
      short_label: "BETA" # Used in CMS when side menu is collapsed
      color: "#2650ea"
      description: "Beta preview site"
```
and after the upgrade they would be missing the definitions for Local / Prod and get grey labels.

Reply:
see #7